nexus/pkg/server/server.go

```diff
@@ -11,6 +11,8 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
+const BufferSize = 0
```

Comment:
Do you want to keep it at 0?

Reply:
nope i have a follow up PR that fixes this... maybe we can close this one